docs/user/convert/function_usage.rst

```diff
@@ -643,7 +643,7 @@ delimiters for a map:
 
 ::
 
-    { name = "numbers", transform = "parseMap('int -> string', $2, ',', '->')" }
+    { name = "numbers", transform = "parseMap('int -> string', $2, '->', ',')" }
```

Comment:
Is this an API change:)?

Reply:
haha, nope... the example was just wrong :)